src/components/mini-widgets/MiniVideoRecorder.vue

```diff
@@ -144,8 +144,9 @@ watch(nameSelectedStream, () => {
 
 // Fetch temporary video data from the storage
 const fetchNumebrOfTempVideos = async (): Promise<void> => {
-  const size = await videoStore.videoStoringDB.length()
```

Comment:
This line has changed slightly recently, so probably is going to conflict with the pr #874 changes.